Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -92,9 +92,17 @@ open class Container: UIObject {
     func addPlugin(_ plugin: UIContainerPlugin) {
         plugins.append(plugin)
     }
+    
+    private func findPlugin(_ pluginClass: AnyClass) -> [UIContainerPlugin] {
+        return plugins.filter{ $0.isKind(of: pluginClass) }
```

Comment:
Instead of use filter and return an array. Isn't better to use first as first is faster than filter (as it stops the loop when find the first match) and is more legible because (for me) a method with the name `findPlugin` has to return an element or nil instead of an array.